.github/workflows/publish-docfx.yml

```diff
@@ -0,0 +1,39 @@
+
+name: Deploy DocFx
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
```

Comment:
Can we limit this to run only when documentation files are actually changed? We don't want to unnecessarily push a new commit for every non-docs change.

Reply:
Added paths/ check.